src/auth/passport.js

```diff
@@ -24,15 +27,15 @@ passport.use(
       User.findOne({ email, role: 'regular' })
         .then(result => {
           if (!result) {
-            throw new ValidationError({ password: 'Няправільны емэйл ці пароль' });
+            throw new ValidationError({ email: dictionary.be.badEmail });
```

Comment:
TODO: implement error codes, and move this dictionary on client dise

Reply:
shall we introduce our own error coding system or to use an existing one? shall we send numbers or text?

Reply:
I imagined this like strings ids (`ERR_EMAIL`, `ERR_PASSW`). About existing system - we should research this question.